app/components/ui/create-tale-modal/component.js

```diff
@@ -25,9 +26,26 @@ export default Component.extend({
   }),
   
   defaultErrorMessage: "There was an error while creating your Tale.",
+  
+  didRender() {
+    const component = this;
+    $('#as-tale-false-chkbox').checkbox({
+      onChecked: function() {
+        console.log('Toggling asTale to false');
```

Comment:
Can we drop `console.log` here and a few lines below?

Reply:
Done - `console.log` lines have been dropped